vitrina/requests/views.py

```diff
@@ -14,3 +17,47 @@ class RequestListView(ListView):
 class RequestDetailView(DetailView):
     model = Request
     template_name = 'vitrina/requests/detail.html'
+
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        request = context_data.get('request')
+
+        structure_data = request.structure_data.split(";") if request.structure_data else []
+        structure = []
+        for struct in structure_data:
+            data = struct.split(",")
+            if data:
+                structure.append({
+                    "data_title": data[0],
+                    "dictionary_title": data[1],
+                    "data_type": data[2],
+                    "data_notes": data[3],
+                })
+
+        dataset = None
+        if request.dataset_id:
+            try:
+                dataset = Dataset.public.get(pk=request.dataset_id)
+            except ObjectDoesNotExist:
+                pass
+
+        liked = False
+        if self.request.user and self.request.user.pk:
```

Comment:
```suggestion
        if self.rrequest.user.is_authenticated:
```

https://docs.djangoproject.com/en/3.2/topics/auth/default/#authentication-in-web-requests